cmd/gcsim/main.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/genshinsim/gcsim/pkg/calcqueue"
 	"github.com/genshinsim/gcsim/pkg/core"
 	"github.com/genshinsim/gcsim/pkg/parse"
+	_ "github.com/montanaflynn/stats"
```

Comment:
not sure this is needed here? stats should be calculated in the sim anyways not in the cmd line?

Reply:
Removed the import